src/app/public/src/modules/shared/omnibar-adapter.service.ts

```diff
@@ -25,7 +27,24 @@ export class StacheOmnibarAdapterService {
   }
 
   private setHeight(): void {
-    this.omnibarHeight = this.element.offsetHeight;
+    let omnibarElementHeight = this.element.offsetHeight;
+
+    if (this.omnibarIsExpanded(omnibarElementHeight)) {
+
+      if (this.setHeightAttempts < 10) {
```

Comment:
Nice! Are we going to run this check for 5 seconds? Is that too long? I'm not sure personally, but that feels like a long time to wait for a load. But maybe it's better to be safe. It's not like this is an expensive operation.

Reply:
It' won't delay loading. All this will do is give it a max of 5 seconds for the omnibar to finish collapsing back to it's standard size before saying "forget it"  and moving on.

Reply:
Right, I just meant it was a long time to wait for the omnibar to load. But honestly, it probably won't be much of an issue.